src/ctapipe/containers.py

```diff
@@ -773,12 +775,11 @@ class SimulatedCameraContainer(Container):
     )
 
 
-class SimulatedEventContainer(Container):
+class SimulationSubarrayContainer(Container):
     shower = Field(
         default_factory=SimulatedShowerContainer,
         description="True event information",
     )
-    tel = Field(default_factory=partial(Map, SimulatedCameraContainer))
```

Comment:
With the tel removed this container now only contains one thing, so its not clear what the extra indirection step achieves.